packages/office-ui-fabric-react/src/components/ProgressIndicator/ProgressIndicator.types.ts

```diff
@@ -17,18 +19,28 @@ export interface IProgressIndicatorProps {
   /**
    * Label to display above the control.
    */
-  label?: string;
+  label?: React.ReactNode;
```

Comment:
This is a breaking API change. @dzearing  what is our API change protocol?

Reply:
`React.ReactNode` includes `string` as a valid type. This should not be breaking.